testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -3188,6 +3190,33 @@ QUnit.module('searchEnabled', moduleSetup, () => {
         assert.ok($input.width() > inputWidth, 'input size is changed for substitution');
     });
 
+    QUnit.test('the previous value should be still selected after the new value was added after the new search (T880346)', function(assert) {
+        const items = ['aaa', 'aab', 'bbb'];
+        const $element = $('#tagBox').dxTagBox({
+            dataSource: items,
+            searchEnabled: true,
+            applyValueMode: 'useButtons',
+            searchTimeout: 0,
+            showSelectionControls: true,
+            deferRendering: true,
+            minSearchLength: 2
+        });
+        const instance = $element.dxTagBox('instance');
+        const $input = $element.find(`.${TEXTBOX_CLASS}`);
+
+        keyboardMock($input)
+            .type('aa');
+        $(instance.content()).find(`.${LIST_CKECKBOX_CLASS}`).eq(0).trigger('dxclick');
```

Comment:
```suggestion
        $(`.${LIST_CKECKBOX_CLASS}`).eq(0).trigger('dxclick');
```
Why not like [here](https://github.com/DevExpress/DevExtreme/pull/12874/files#diff-2da7b06a8e2cf07b9cb96e242b9fba75R3214)?